packages/wonder-blocks-button/src/components/button.md

```diff
@@ -420,12 +420,47 @@ const styles = StyleSheet.create({
     </View>
 </MemoryRouter>
 ```
+### Running callbacks on navigation
+
+Sometimes you may need to run some code and also navigate when the user
+clicks the button. For example, you might want to send a request to the
+server and also send the user to a different page. You can do this by
+passing in a URL to the `href` prop and also passing in a callback
+function to either the `onClick`, `beforeNav`, or `safeWithNav` prop.
+Which prop you choose depends on your use case.
```

Comment:
Only passing one of these callbacks is super important so it might warrant calling this out with a special note or warning.

Reply:
Can you say more about this? The Flow types explicitly allow any combination of the 3 props, and code comments document what happens if you pass both `beforeNav` and `safeWithNav`. Why is it important to only pass one of them?

Reply:
I dug into the code some and determined that it is indeed safe for multiple callbacks to be used together, then updated the documentation with the details of how they interact.